client/src/components/product-filter/index.js

```diff
@@ -273,6 +281,24 @@ const ProductFilter = ({
     }
   };
 
+  const handlePlatformClick = () => {
+    platformList ? setPlatformList(false) : setPlatformList(true); 
+  };
+
+  const handleCategoryClick = () => {
+    categoriesList ? setCategoriesList(false) : setCategoriesList(true); 
+  };
+
+  const handleStoreClick = () => {
+    storeList ? setStoreList(false) : setStoreList(true); 
+  };
+
+  const handlePriceClick = () => {
+    priceList ? setPriceList(false) : setPriceList(true); 
+  };
+
```

Comment:
For the handler functions, we could proably change it to, for instance,:
`const handlePlatformClick = () => {
    setPlatformList((state) => !state);
}`
state being the current true/false value